index.html

```diff
@@ -8293,13 +8308,13 @@ <h2 class="Annex" id="G-References">
 Video</i>. John Wiley and Sons, Inc., New York, 1996. ISBN
 0-471-12253-X.</dd>
 
-<!-- Maintain a fragment named "G-SMPTE-170M" to preserve incoming links to it -->
-<dt id="G-SMPTE-170M">
-<a name="SMPTE-170M">[SMPTE-170M]</a></dt>
+<!-- Maintain a fragment named "G-SMPTE 170M" to preserve incoming links to it -->
+<dt id="G-SMPTE 170M">
+<a name="SMPTE 170M">[SMPTE 170M]</a></dt>
```

Comment:
The old fragment had a dash and needs to be preserved.
The new fragment (<a name="...") can have it removed if spaces are allowed in fragments. I assume they are?

Reply:
That whole section will go away because it is replaced by the automatic ReSpec generated references.

I'm unsure how to preserve links to old bibliographic entries (apart from hand-editing the generated references to add them back in)

Reply:
Let's preserve it for now. If we end up deleting it as part of the move to ReSpec, that is okay. But let's break it there, not here.

Reply:
This *is* the move to ReSpec :)

Reply:
But okay